pkg/console/subresource/configmap/configmap.go

```diff
@@ -18,23 +19,57 @@ const (
 	consoleConfigYamlFile   = "console-config.yaml"
 	clientSecretFilePath    = "/var/oauth-config/clientSecret"
 	oauthEndpointCAFilePath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-	// TODO: should this be configurable?  likely so.
-	documentationBaseURL = "https://docs.okd.io/4.0/"
-	brandingDefault      = "okd"
 	// serving info
 	certFilePath = "/var/serving-cert/tls.crt"
 	keyFilePath  = "/var/serving-cert/tls.key"
 )
 
-func DefaultConfigMap(cr *operatorv1.Console, rt *routev1.Route) *corev1.ConfigMap {
+// overridden by console config
+const (
+	defaultLogoutURL = ""
+)
+
+// overridden by operator config
+const (
+	defaultDocumentationBaseURL = "https://docs.okd.io/4.0/"
+	defaultBranding             = "okd"
+)
+
+func getLogoutRedirect(consoleConfig *configv1.Console) string {
+	if len(consoleConfig.Spec.Authentication.LogoutRedirect) > 0 {
+		return consoleConfig.Spec.Authentication.LogoutRedirect
+	}
+	return defaultLogoutURL
+}
+
+func getBrand(operatorConfig *operatorv1.Console) operatorv1.Brand {
+	if len(operatorConfig.Spec.Customization.Brand) > 0 {
+		return operatorConfig.Spec.Customization.Brand
+	}
+	return defaultBranding
+}
+
+func getDocURL(operatorConfig *operatorv1.Console) string {
+	if len(operatorConfig.Spec.Customization.DocumentationBaseURL) > 0 {
+		return operatorConfig.Spec.Customization.DocumentationBaseURL
+	}
+	return defaultDocumentationBaseURL
+}
+
+func DefaultConfigMap(operatorConfig *operatorv1.Console, consoleConfig *configv1.Console, rt *routev1.Route) *corev1.ConfigMap {
+
+	logoutRedirect := getLogoutRedirect(consoleConfig)
+	brand := getBrand(operatorConfig)
+	docURL := getDocURL(operatorConfig)
+
 	host := rt.Spec.Host
-	config := NewYamlConfigString(host)
+	config := string(NewYamlConfig(host, logoutRedirect, brand, docURL))
```

Comment:
I eliminated the one wrapper func `NewYamlConfigString()`, was unnecessary additional plumbing for passing these values down, when all it did was call `string()`.